### ISSUE TITLE
License description in README is not actually reflective of MIT license

### ISSUE THREAD

**Issue by TheButlah** [SlimeVR/SlimeVR-Server#170]
The MIT license does not require people to announce that they are using SlimeVR. It merely requires them to provide the original license document (the one with Eiren's copyright) along with the software. This can be accomplished by providing it in the open source codebase if a project is open source, or by providing it as a license in a menu somewhere in some closed source application.

The section in the readme as it stands is therefore confusing and may give legal uncertainty on whether the project is *actually* MIT licensed, or if its an MIT license with some additional requirements, It also causes problems because contributors believe their contributions are MIT licensed not "MIT + additional requirements"

I suggest rewording that section. Maybe something like this:

>## License
>
>SlimeVR software (including server, firmware, drivers, installator, documents, and others - see licence for each case specifically) is distributed under the [MIT License](https://github.com/SlimeVR/SlimeVR-Server/blob/main/LICENSE) and is copyright of Eiren Rain and SlimeVR. The MIT Licence is a permissive license giving you rights to modify and distribute the software with little strings attached.
>
>However, the MIT License has some limits, and if you wish to distribute software based on SlimeVR, you need to be aware of them:
>
>* When distributing any software that uses or is based on SlimeVR, you have to provide to the end-user the original, unmodified `LICENSE` file from SlimeVR. This file is located [here](https://github.com/SlimeVR/SlimeVR-Server/blob/main/LICENSE). This includes the `Copyright (c) 2021 Eiren Rain, SlimeVR` part of the license. It is not sufficient to use a generic MIT License, it must be the original `LICENSE` file.
>* This applies even if you distribute software without the source code. In this case, one way to provide it to the end-user is to have a menu in your application that lists all the open source licenses used, including SlimeVR's.
>
>Please refer to the [LICENSE](https://github.com/SlimeVR/SlimeVR-Server/blob/main/LICENSE) file if you are at any point uncertain what the exact the requirements are.

**Comment by Eirenliel** [comment]
Fixed in https://github.com/SlimeVR/SlimeVR-Server/commit/c74cc539c92e59ec9bc58617309a9401085c718c
